apps/frontend/src/views/Checklist.vue

```diff
@@ -0,0 +1,312 @@
+<template>
+  <Base
+    :show-search="true"
+    :title="currentTitle"
+    @changed-files="evalInfo = null"
+  >
+    <!-- Topbar content - give it a search bar -->
+    <template #topbar-content>
+      <v-btn :disabled="!canClear" @click="clear">
+        <span class="d-none d-md-inline pr-2"> Clear </span>
+        <v-icon>mdi-filter-remove</v-icon>
+      </v-btn>
+      <UploadButton />
+      <ExportButton>
+        <v-list-item>
+          <v-tooltip top>
+            <template #activator="{on}">
+              <IconLinkItem
```

Comment:
the @click seems to have accidentally been deleted.  that function will probably need to be adjusted to deal with the fact that the intermediary format is in the passthrough of the hdf file when david's work gets merged in

Reply:
Added back the @click for `exportCkl()`